### ISSUE TITLE
Bilateral polymicrogyria

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6757]
**Preferred term label:**
Bilateral polymicrogyria

**Synonyms**

**Definition (free text, please give PubMed ID)**
Polymicrogyria that affects all or some of both cerebral hemispheres.

(This is a reasonably frequent finding and not always generalized or perisylvian).
**Parent term (use hpo.jax.org/app)**
Polymicrogyria HP:0002126

Children:
Bilateral generalized polymicrogyria HP:0032410
Bilateral perisylvian polymicrogyria HP:0032407

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
thanks, added new term!
